sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -77,9 +90,9 @@ export class AzureCliCredential implements TokenCredential {
     scopes: string | string[],
     options?: GetTokenOptions
   ): Promise<AccessToken> {
+    this.tenantId = processMultiTenantRequest(this.tenantId, options);
```

Comment:
You shouldn't assign the result of `processMultiTenantRequest` back to `this.tenantId`. The `tenantId` requested in `GetTokenOptions` is ephemeral and should only impact that specific request, not mutate the credential state.

Reply:
Oh thank you!